app/services/push_notification/http_push.rb

```diff
@@ -19,6 +19,8 @@ def initialize(event, now: Time.zone.now)
     def deliver
       return unless IdentityConfig.store.push_notifications_enabled
 
+      # mattw: I think we should change this but this breaks tests, possibly because
+      # the test calls user.identities and we need to change that still?
       event.user.
         service_providers.
         merge(ServiceProviderIdentity.not_deleted).
```

Comment:
```suggestion
      event.user.
        service_providers.
        merge(ServiceProviderIdentity.not_deleted).
        merge(ServiceProviderIdentity.consented).
```

yes definitely should be here, we need to fix the tests